{"name": "test_4[chromium]", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nCall log:\nnavigating to \"https://skillbox.ru/code/\", waiting until \"domcontentloaded\"", "trace": "self = <module_7.TestValidate object at 0x00000267C01295E0>\npage = <Page url='about:blank'>\ngo_to_url = <function go_to_url.<locals>.callback at 0x00000267C1383CE0>, x = 0\ny = 0\n\n    def test_4(self, page, go_to_url, x=0, y=0):\n        # Шаг 1: Перейдите на сайт Skillbox\n>       go_to_url('https://skillbox.ru/code/')\n\nmodule_7.py:87: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\src\\fixtures\\system\\browser.py:49: in callback\n    page.goto(url, wait_until='domcontentloaded')\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:8843: in goto\n    self._sync(\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_page.py:524: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_frame.py:145: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright._impl._connection.Connection object at 0x00000267C01290A0>\ncb = <function Channel.send.<locals>.<lambda> at 0x00000267C13839C0>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nE           Call log:\nE           navigating to \"https://skillbox.ru/code/\", waiting until \"domcontentloaded\"\n\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_connection.py:514: TimeoutError"}, "steps": [{"name": "Открытие страницы 'https://skillbox.ru/code/'", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nCall log:\nnavigating to \"https://skillbox.ru/code/\", waiting until \"domcontentloaded\"\n\n", "trace": "  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\PycharmProjects\\pythonProject\\src\\fixtures\\system\\browser.py\", line 49, in callback\n    page.goto(url, wait_until='domcontentloaded')\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 8843, in goto\n    self._sync(\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_page.py\", line 524, in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_frame.py\", line 145, in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 59, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 514, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "parameters": [{"name": "url", "value": "'https://skillbox.ru/code/'"}], "start": 1734966414514, "stop": 1734966444554}], "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1734966414513, "stop": 1734966444559, "uuid": "437da046-f7a0-4998-b44e-9ce91ffc46fd", "historyId": "9808811551ec276a6f0c854eb9118b9b", "testCaseId": "2625270973a6db0ba7dc8f0ee0858324", "fullName": "Tests.homework.module_7.TestValidate#test_4", "labels": [{"name": "parentSuite", "value": "Tests.homework"}, {"name": "suite", "value": "module_7"}, {"name": "subSuite", "value": "TestValidate"}, {"name": "host", "value": "Admin"}, {"name": "thread", "value": "12956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.homework.module_7"}]}